### PR TITLE
Update store-apps-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the navigation

fixes https://github.com/MicrosoftDocs/memdocs/issues/4212

### CODE REVIEW

File: memdocs/intune/apps/store-apps-windows.md [hunk 1]
```diff
@@ -45,16 +45,14 @@ You can add a Microsoft Store app to Intune by doing the following:
...
 4. Click **Select**. The **Add app** steps are displayed.
 5. To configure the **App information** for Microsoft store apps, click **Select app**, and search for the app you want to assign to members of your organization. Display the app page and make a note of the app details. 
 6. In the **App information** page, add the app details:
     - **Name**: Enter the name of the app as it is to be displayed in the company portal. Make sure that any app name that you use is unique. If an app name is duplicated, only one name is displayed to users in the company portal.
     - **Description**: Enter a description for the app. This description is displayed to users in the company portal.
     - **Publisher**: Enter the name of the publisher of the app. 
-    - **Package Identifier**: The app Package Identifier is the unique value that identifies the app.
-    - **Installer Type**: The installer type of the application package.
-	- **Install behavior**: Select **System** to install this app for all users, if supported. Select **User** to install this app for the logged-in user on the device.
+    - **Appstore URL**: Enter the 'Link for Intune' URL for the app provided from the store.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/store-apps-windows.md:55]:
```suggestion
    - **Appstore URL**: Enter the 'Link for Intune' URL for the app provided by the store.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and signoff this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**